tests/robot-tests/tests/libs/admin_api.py

```diff
@@ -161,6 +162,39 @@ def user_adds_publication_role_to_user_via_api(
         f'Adding publication role to user API request failed with {response.status_code} and {response.text}'
 
 
+def user_removes_all_release_and_publication_roles_from_user(user_id: str) -> None:
+    response = admin_client.delete(
+        f'/api/user-management/user/{user_id}/release-publication-roles/all'
```

Comment:
Again in line with my other comments, I'd update this URL to be `resource-roles` rather than `release-publication-roles`.